spotifycli/__main__.py

```diff
@@ -1,2 +1,3 @@
+import sys
 from .spotifycli import main
-main()
+sys.exit(main())
```

Comment:
The `main()` function now returns a meaningful exit code, so we can use that when running `spotifycli` on the command line through this wrapper and give the user the meaningful exit code.